app/scripts/lib/auth-errors.js

```diff
@@ -187,6 +187,10 @@ function (_, Errors) {
     INVALID_EMAIL: {
       errno: 1023,
       message: t('Valid email required')
+    },
+    FORCE_AUTH_EMAIL_REQUIRED: {
```

Comment:
I'd favor a mix of two and three: Show "Unexpected error" then perhaps a hint to developers to check the console for more info.

> A third alternative is to use the developer console. Right now our console is pretty noisy, if reduce the amount of noise we could give RP devs information they can use.

Any details on the noise you're seeing? I filed #2036, which was the only noise I noticed while causally perusing the site.